onflow/docs
# Nialexsan/mixpanel

Added mixpanel integration
https://github.com/onflow/developer-portal/issues/983

## Testing

- add `MIXPANEL_PROJECT_TOKEN` to .env file located in 1Password vault -> `.env Flow Docusaurus Prod`
- run project locally and open any doc
- check `Flow - Developer Portal` dashboard in mixpanel
- you should see a `Page View` event

<img width="1348" alt="Screen Shot 2023-04-11 at 11 12 03 AM" src="https://user-images.githubusercontent.com/12097569/231208502-c56fe230-4335-4a5f-9237-693e210cec6a.png">
